docs/general/server/media/shows.md

```diff
@@ -68,6 +68,14 @@ Episode numbering for specials may vary from metadata provider to metadata provi
 
 :::
 
+Specials can also be shown within a Season if so desired. This can be helpful when they are part of a continued storyline during the Season. This requires 2 settings:
+
+1. The option under Dashboard -> Library -> Display -> "Display specials within their series they aired in" must be enabled
+2. The Season and Episode they aired before/after must be set within the metadata.
+    * These can be set in the Metadata editor or in an NFO using the `airsbefore_season`, `airsafter_season`, and `airsbefore_episode` tags.
```

Comment:
Lists should use a dash (`-`)

Reply:
In that case the Linter should probably be adjusted because the CI run is failing because dash was used everywhere else and the Linter expects asterisks...

I thought about fixing that up while I'm already touching the file but left it for now.

Reply:
Ah, bit of a misleading message from the Linter then, but oh well, it's fixed.